sql/core/src/test/scala/org/apache/spark/sql/CharVarcharTestSuite.scala

```diff
@@ -603,6 +611,25 @@ class FileSourceCharVarcharTestSuite extends CharVarcharTestSuite with SharedSpa
       }
     }
   }
+
+  test("DESCRIBE COLUMN w/ char/varchar") {
```

Comment:
shall we add a TODO? eventually, v2 catalog can run these 2 commands as well.

Reply:
Any existing JIRA to track this, or I shall create one, then mark todo w/ it togeter

Reply:
https://issues.apache.org/jira/browse/SPARK-33875 found it